day1/src/main.rs

```diff
@@ -0,0 +1,53 @@
+use core::fmt::Display;
+use std::fs;
+
+enum Digits {
+    One,
+    Two,
+    Three,
+    Four,
+    Five,
+    Six,
+    Seven,
+    Eight,
+    Nine,
+}
+
+impl Display for Digits {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            Digits::One => write!(f, "one"),
+            Digits::Two => write!(f, "two"),
+            Digits::Three => write!(f, "three"),
+            Digits::Four => write!(f, "four"),
+            Digits::Five => write!(f, "five"),
+            Digits::Six => write!(f, "six"),
+            Digits::Seven => write!(f, "seven"),
+            Digits::Eight => write!(f, "eight"),
+            Digits::Nine => write!(f, "nine"),
+        }
+    }
+}
+
+fn main() {
+    let contents = fs::read_to_string("./input.txt").unwrap();
+    let splited = contents.split("\n");
```

Comment:
Більш ідіоматично використати тут `lines()`

```suggestion
    let splited = contents.lines();
```